tools/mobrpg/mobrpg/commands/images.py

```
@@ -0,0 +1,142 @@
+from mobrpg import client
+from mobrpg import vault as _vault
+
+KINDS = ["person", "organization", "political", "landfeature", "item"]
+FOLDER = {"person": "characters", "organization": "factions",
+          "political": "locations", "landfeature": "locations", "item": "items"}
+
+
+def _download(url: str) -> bytes:
+    """Fetch raw bytes from a URL. Factored out so tests can stub the network
+    without ever hitting a live server."""
+    with urllib.request.urlopen(url, timeout=30) as resp:
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)